tap_zendesk/streams.py

```diff
@@ -114,13 +115,33 @@ def load_metadata(self):
     def is_selected(self):
         return self.stream is not None
 
+    def check_access(self):
+        '''
+        Check whether permission given to access stream resource or not.
```

Comment:
Check whether the permission was given to access stream resources or not.
```suggestion
        Check whether permission given to access stream resource or not.
```

Reply:
Updated.